### PR TITLE
[11.0] add stock_orderpoint_move_link and stock_orderpoint_purchase_link

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Stock Orderpoint Purchase Link
==============================

This module adds to purchase order lines a direct link to the reordering rules
that created them. In chained moves, the reordering rule is propagated
from stock moves to the purchase order line.

Stock Orderpoint Move Link
==============================

This module adds to stock moves a direct link to the reordering rules
that created them. In chained moves, the reordering rule is propagated.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@bodedra This is what I meant, not https://github.com/OCA/stock-logistics-warehouse/pull/432

**Comment by JordiBForgeFlow** [comment]
cc @max3903

**Comment by bodedra** [comment]
Please review patch at https://github.com/Eficent/stock-logistics-warehouse/pull/15

**Comment by bodedra** [comment]
It seems have two good suggestion. What will be best approaches ?

#435 OR https://github.com/OCA/stock-logistics-workflow/pull/425 

CC @max3903 @grindtildeath  @jbeficent @rousseldenis @simahawk

**Comment by JordiBForgeFlow** [comment]
To me they can live together. They address different issues.

File: stock_orderpoint_purchase_link/views/purchase_order_views.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<!-- Copyright 2016 Eficent Business and IT Consulting Services S.L.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
```
Comment (bodedra) [inline stock_orderpoint_purchase_link/views/purchase_order_views.xml:10]: We can remove these if nothing to put inside.

**Review by bodedra — APPROVED** [review]
Code review. :+1:

**Comment by JordiBForgeFlow** [comment]
I will add more tests.

**Review by max3903 — APPROVED** [review]

**Review by jgrandguillaume — APPROVED** [review]
LGTM

**Comment by JordiBForgeFlow** [comment]
I will add tests in a separate PR

**Pull request merged**